test/csuite/wt2834_join_bloom_fix/main.c

```diff
@@ -161,7 +161,7 @@ main(int argc, char *argv[])
 	return (0);
 }
 
-void *populate(TEST_OPTS *opts)
+void populate(TEST_OPTS *opts)
```

Comment:
These changes are definitely arguable.  The reason I touched this file is that I saw `WT_ROLLBACK` returned from the `WT_CURSOR::insert` call, which the test code doesn't expect.  Using an auto-commit `insert` avoids the issue (WT loops internally on rollback).

I don't know for sure if it relies on the `search` and `insert` operations have to happen in a transaction for correctness of the test, and I don't think the `WT_ROLLBACK` will happen with the current state of the branch.